backend/python/app/rest/entity_routes.py

```diff
@@ -70,7 +95,17 @@ def create_entity():
 @validate_request("EntityDTO")
 def update_entity(id):
     try:
+        # file-storage {
+        if request.content_type == "application/json":
+            body = EntityDTO(**request.json)
+        else:
+            req = json.loads(request.form.get("body"))
+            req_file = request.files.get("file", default=None)
```

Comment:
You could change this to the below for consistency if you want
```
req["file"] = request.files.get("file", default=None)
            body = EntityDTO(**req)
```
the two are equivalent. I just realized that @tamim-khan wrote the all the other routes differently from this one, which I wrote.

Reply:
I'll make note to change this in `main` branch first, leaving this for now